### PR TITLE
Add spacefish prompt

### PR DESCRIPTION
**Pull request by matchai**

### CODE REVIEW

File: index [hunk 1]
```diff
@@ -484,6 +484,12 @@ Heavily inspired by @mathiasbynens's prompt
 prompt simple
 sotayamashita
 
+spacefish
+https://github.com/matchai/spacefish
+🚀🐟 A fish port of spaceship
```
Comment (laughedelic) [inline index]: It sounds cool, but for clarity I'd add "prompt" at the end: `🚀🐟 A fish port of spaceship prompt`

Comment (laughedelic) [reply index]: Or even better (IMO) use your headline:

> A Fish Shell prompt for Astronauts

Comment (matchai) [reply index]: Good call! I'll go ahead and change it.

**Comment by JorgeBucaran** [comment]
🎉

**Pull request merged**